pyaerocom/aeroval/experiment_processor.py

```diff
@@ -122,7 +122,11 @@ def run(self, model_name=None, obs_name=None, var_list=None, update_interface=Tr
         if not self.cfg.model_cfg:
             logger.info("No model found, will make dummy model data")
             self.cfg.webdisp_opts.hide_charts = ["scatterplot"]
-            self.cfg.webdisp_opts.hide_pages = ["maps.php", "intercomp.php", "overall.php"]
+            self.cfg.webdisp_opts.hide_pages = [
+                "maps.php",
+                "intercomp.php",
+                "overall.php",
+            ]
```

Comment:
Do we still write .php pages? I thought @AugustinMortier had removed those?

Reply:
I have no idea but this is main-dev code, so I'd rather address that somewhere else than in this PR

Reply:
ok